notify/email/email.go

```diff
@@ -112,77 +112,87 @@ func (n *Email) auth(mechs string) (smtp.Auth, error) {
 
 // Notify implements the Notifier interface.
 func (n *Email) Notify(ctx context.Context, as ...*types.Alert) (bool, error) {
+	// TODO: move the check to the config pacakge.
```

Comment:
```suggestion
	// TODO: move the check to the config package.
```

Is this supposed to happen in this pull request or a follow up?

Reply:
I planned for a follow up PR.